app/build.gradle

```diff
@@ -14,8 +14,8 @@ android {
         applicationId "ca.bc.gov.vaxcheck"
         minSdk 27
         targetSdk 31
-        versionCode 11
-        versionName "1.0"
+        versionCode 12
```

Comment:
I feel like we're gonna run into problems if we can't auto-increment this on each build. How hard would this be to add?

Reply:
It's not difficult at all. The only thing I see is if we do auto-increment then we will lose control of the version name and code we publish on the play store.